.github/workflows/publish.yml

```diff
@@ -0,0 +1,65 @@
+# GHA workflow recipe to upload repository content using WebDAV when changed.
+# In this setup, the workflow will connect to a tailnet first.
+# https://github.com/tailscale/github-action
+---
+name: Publish content
+
+on:
+  push:
+    branches: [ main ]
+
+  # Only able for testing purposes.
+  # pull_request: ~
+
+  # Allow job to be triggered manually.
+  workflow_dispatch:
+
+# Cancel in-progress jobs when pushing to the same branch.
+concurrency:
+  cancel-in-progress: true
+  group: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+
+  publish-webserver:
+
+    runs-on: "ubuntu-latest"
+
+    name: Upload content to web server
+    steps:
+
+    - name: Connect using Tailscale
+      uses: tailscale/github-action@v3
+      with:
+        oauth-client-id: ${{ secrets.TS_OAUTH_CLIENT_ID }}
+        oauth-secret: ${{ secrets.TS_OAUTH_SECRET }}
+        tags: tag:ci
+
+    - name: Probe network connectivity
+      run: ping -c2 web &
+
+    - name: Acquire sources
+      uses: actions/checkout@v4
+
```

Comment:
Would be cool to convert README.md to README.html after acquiring the sources, so people can read it easily per https://cdn.crate.io/downloads/datasets/cratedb-datasets/README.html.